modules/@apostrophecms/util/ui/src/util.js

```diff
@@ -183,25 +196,37 @@ export default () => {
   // Your player is guaranteed to run only once per widget. Hint:
   // DON'T try to find all the widgets. DO just enhance `el`.
   // This is a computer science principle known as "separation of concerns."
+  //
+  // The second argument is an options object. If the `init` option is true, only
```

Comment:
Sure, but the name is quite confusing. I suggest that you call this option `newPlayersOnly` and have it filter when set to `true`. I also suggest this language:

"If this option is `true`, only widget player types whose players have not been invoked before will be invoked."